jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -1,38 +1,49 @@
 pipeline {
-    agent none
+    agent any
 
     parameters {
         // https://datagrepper.engineering.redhat.com/id?id=ID:messaging-devops-broker02.web.prod.ext.phx2.redhat.com-44775-1580379079663-8:893046:0:0:1&is_raw=true&size=extra-large
         string defaultValue: '', description: 'This will be populated by the CI trigger', name: 'CI_MESSAGE', trim: true
     }
-
     stages {
 
         stage('Process UMB Message') {
             steps {
-                //ToDo Process the input UMB Message (CI_MESSAGE) to extract information required by the following stages
-                echo "Process UMB Message"
+                script{
+                    final Map msg = readJSON text: CI_MESSAGE
```

Comment:
There's something wrong with how I'm trying to do this. Will revisit it later. https://integreatly-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/view/Delorean%202.0/job/delorean-2-ews/job/3scale/job/discovery/18/console